mountpoint-s3/tests/fuse_tests/fork_test.rs

```diff
@@ -380,6 +384,130 @@ fn mount_scoped_credentials() -> Result<(), Box<dyn std::error::Error>> {
     Ok(())
 }
 
+#[cfg(all(feature = "sse_kms", not(feature = "s3express_tests")))]
+fn mount_with_sse(
+    bucket: &str,
+    mount_point: &Path,
+    prefix: &str,
+    key_id: &str,
+    credentials: Option<aws_sdk_s3::config::Credentials>,
+) -> Child {
+    let region = get_test_region();
+    let mut cmd = Command::cargo_bin("mount-s3").expect("can not locate mount-s3 binary");
+    cmd.stdout(Stdio::piped())
+        .arg(bucket)
+        .arg(mount_point)
+        .arg(format!("--region={region}"))
+        .arg(format!("--prefix={prefix}"))
+        .arg("--sse=aws:kms:dsse")
+        .arg(format!("--sse-kms-key-id={key_id}"))
+        .arg("--auto-unmount")
+        .arg("--foreground");
+    if let Some(credentials) = credentials {
+        cmd.env("AWS_ACCESS_KEY_ID", credentials.access_key_id())
+            .env("AWS_SECRET_ACCESS_KEY", credentials.secret_access_key())
+            .env("AWS_SESSION_TOKEN", credentials.session_token().unwrap());
+    }
+    let child = cmd.spawn().expect("unable to spawn child");
+    wait_for_mount("mountpoint-s3", mount_point.to_str().unwrap());
+    child
+}
+
+#[cfg(all(feature = "sse_kms", not(feature = "s3express_tests")))]
+fn erroneous_write(mount_point: &Path) {
+    let mut f = fs::File::create(mount_point.join("file.txt")).expect("can not open file for write");
```

Comment:
nit: these `expect` messages should state why you expected the value to be `Ok`/`Some`.

```suggestion
    let mut f = fs::File::create(mount_point.join("file.txt")).expect("should be able to open file for writing");
```